docs/index.rst

```diff
@@ -3,26 +3,37 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+========
 Welcome!
 ========
 
-Welcome to the Contributors Portal for Access to Memory (AtoM), Archivematica and Enduro.
+Welcome to the Contributors Portal for Access to Memory (AtoM),
+Archivematica and Enduro.
+
+At `Artefactual <https://www.artefactual.com>`__, we are currently
+working on improving the contribution process, and we deemed useful
+to have a Portal to regroup relevant information for our main
+software projects.
 
-This Portal was first created in May 2024.
+This Portal was first created in May 2024. It is maintained as a 
+`GitHub repo <https://www.github.com/artefactual/artefactual.github.io>`__.
+
+Suggestions for this Portal are welcome - you can either `open an issue <https://www.github.com/artefactual/artefactual.github.io/issues>`__
+or `send us an email <mailto:contributors@artefactual.com>`__.
 
 .. toctree::
    :maxdepth: 1
    :caption: Contents:
    
    cheatsheet.rst
+   dependencies.rst
    i18n.rst
    
-Why this Portal
----------------
-
-At Artefactual, we are currently working on improving the contribution process, and we deemed useful to have a Portal to help regroup relevant information for our main software projects.
+The :doc:`Cheatsheet <cheatsheet>`, is a table of links for each
+of the projects we maintain.
 
-What is the Portal
-------------------
+The page :doc:`Dependencies <dependencies>` is a set of notes on
+dependencies of the projects we maintain.
 
-The Portal is a way for Contributors to find information that will guide them through the contribution process for AtoM, Archivematica and Enduro.
+The :doc:`Internationalization <i18n>` page contains information related to
+the current state of the translation process for AtoM and for Archivematica.
```

Comment:
not necessary, but could just say "our projects" rather than listing AtoM and Archivematica - hopefully we will add i18n support for Enduro in the future, and/or we will list other projects on that page, and this way you won't have to update this index page.